openshift-kube-apiserver/openshiftkubeapiserver/sdn_readyz_wait.go

```diff
@@ -93,54 +73,28 @@ func (c *aggregatedAPIServiceAvailabilityCheck) checkForConnection(context gener
 			},
 			Timeout: 1 * time.Second, // these should all be very fast.  if none work, we continue anyway.
```

Comment:
1 second is tooo short in production environments

Reply:
Hi, part of the code which do a http request is not changed, the 1 second value is used in current releases, should that be changed in your opinion?

Reply:
yeah, I just noticed now, forget the comment, I really understand it well